.meepctl-repocfg.yaml

```diff
@@ -573,6 +579,79 @@ repo:
+      meep-edge-platform-app-enablement:
+        # location of source code
+        src: go-apps/meep-edge-platform-app-enablement
+        # location of binary
+        bin: bin/meep-edge-platform-app-enablement
+        # location of deployment chart
+        chart: charts/meep-edge-platform-app-enablement
+        # user supplied value file located @ .meep/user/values (use below file name)
+        chart-user-values: meep-edge-platform-app-enablement.yaml
+        # extra build flags
+        build-flags:
+          - -mod=vendor
+        # enable meepctl build
+        build: true
+        # enable meepctl dockerize
+        dockerize: true
+        # enable meepctl deploy/delete
+        deploy: true
+        # supports code coverage measurement when built in codecov mode
+        codecov: false
+        # supports linting
+        lint: true
+        # location of API specification
+        api: go-apps/meep-edge-platform-app-enablement/api/app-support/swagger.yaml
+      meep-app-support:
+        ## location of source code
+        src: go-apps/meep-edge-platform-app-enablement/server/app-support
+        ## location of binary
+        #bin: bin/meep-edge-platform-app-enablement
+        ## location of deployment chart
+        #chart: charts/meep-edge-platform-app-enablement
+        ## user supplied value file located @ .meep/user/values (use below file name)
+        #chart-user-values: meep-edge-platform-app-enablement.yaml
+        # extra build flags
+        #build-flags:
+        #  - -mod=vendor
+        # enable meepctl build
+        build: false
+        # enable meepctl dockerize
+        dockerize: false
+        # enable meepctl deploy/delete
+        deploy: false
+        # supports code coverage measurement when built in codecov mode
+        codecov: false
+        # supports linting
+        lint: true
+        # location of API specification
+        api: go-apps/meep-edge-platform-app-enablement/api/app-support/swagger.yaml
+      meep-service-mgmt:
+        # location of source code
+        src: go-apps/meep-edge-platform-app-enablement/server/service-mgmt
+        ## location of binary
+        #bin: bin/meep-edge-platform-app-enablement
+        ## location of deployment chart
+        #chart: charts/meep-edge-platform-app-enablement
+        ## user supplied value file located @ .meep/user/values (use below file name)
+        #chart-user-values: meep-edge-platform-app-enablement.yaml
+        # extra build flags
+        #build-flags:
+        #  - -mod=vendor
+        # enable meepctl build
+        build: false
+        # enable meepctl dockerize
+        dockerize: false
+        # enable meepctl deploy/delete
+        deploy: false
+        # supports code coverage measurement when built in codecov mode
+        codecov: false
+        # supports linting
+        lint: true
+        # location of API specification
+        api: go-apps/meep-edge-platform-app-enablement/api/service-mgmt/swagger.yaml
```

Comment:
meep-app-support & meep-service-mgmt should be removed because they are included in the meep-app-enablement microservice (go-app). To do so you will need to support a list of API files, which will also require a meepctl update.

Reply:
updated :
- api is now a map string in .meep_repocfg file
- no mention of the 2 apis within the single app-enablement api other than in the api map string
- meepctl build updated accordingly

tested and pod comes up with swagger ui updated properly